packages/config/src/proxy.js

```diff
@@ -54,6 +63,14 @@ function createInsightsProxy({ betaEnv, rootFolder, localChrome, customProxy = [
                     }
                 }
             }] : []),
+            ...proxyRoutes ? Object.entries(proxyRoutes).map(([ route, redirect ]) => ({
+                context: (path) => path.includes(route),
+                target: redirect.host || redirect === 'PORTAL_BACKEND_MARKER' ? target : redirect.host || redirect,
+                secure: false,
+                changeOrigin: true,
+                autoRewrite: true,
+                ws: true
+            })) : [],
```

Comment:
If the route is an object, I would spread it here.

Reply:
Good point!